hbase/stackable/patches/hbase/2.4.17/006-HBASE-jruby-version-bump.patch

```diff
@@ -0,0 +1,11 @@
+--- a/pom.xml	2023-03-31 20:03:27.000000000 +0200
++++ b/pom.xml	2024-05-14 15:52:58.901324926 +0200
+@@ -576,7 +576,7 @@
+     <wx.rs.api.version>2.1.1</wx.rs.api.version>
+     <glassfish.jsp.version>2.3.2</glassfish.jsp.version>
+     <glassfish.el.version>3.0.1-b08</glassfish.el.version>
+-    <jruby.version>9.2.13.0</jruby.version>
++    <jruby.version>9.4.7.0</jruby.version>
```

Comment:
Have we tested this? [According to Wikipedia](https://en.wikipedia.org/wiki/JRuby#Release_history), this means effectively upgrading from Ruby 2.5.7 to 3.1. Things may have changed, but AFAIK, Ruby's versioning is so loose that even patch level bumps are sometimes incompatible.

Reply:
I ran the integration tests in the hbase-operator, which at least succeeded. Are there any other ways to test this? I'm not familiar with Ruby or how it is used by HBase.

Reply:
I'm not familiar with where it's used either.. @lfrancke might have more of a clue?

Not sure about hbase, but most of our integration tests don't do a great job exercising the products beyond "does it launch? can I connect?". :/

Reply:
Yeahh, that was always broken for some reason that I still haven't had the energy to investigate properly.

Reply:
You could also give the [hbase-hdfs-load-cycling-data demo](https://docs.stackable.tech/home/stable/demos/hbase-hdfs-load-cycling-data) a try

Reply:
JRuby is used in the HBase shell only as far as I know.
We might be using that in tests etc. so it's definitely worth checking whether that still works properly.